jerry-core/ecma/operations/ecma-regexp-object.h

```diff
@@ -53,7 +55,7 @@ typedef struct
 } re_matcher_ctx_t;
 
 ecma_value_t ecma_op_create_regexp_object_from_bytecode (re_compiled_code_t *bytecode_p);
-ecma_value_t ecma_op_create_regexp_object (ecma_string_t *pattern_p, ecma_string_t *flags_str_p);
+ecma_value_t ecma_op_create_regexp_object (ecma_string_t *pattern_p, uint16_t flags);
```

Comment:
`uint16_t` -> `uint8_t`

Reply:
I don't see any improvements with this change, and changing it would mean changing in other functions since the original `re_parse_regexp_flags` function uses an `uint16_t` pointer as the output value.

Reply:
3 bits are enough for RegExp flags, uint16_t looks an overkill for this, but it looks unrelated to this PR. It can be done a followup PR.